test/src/api/responderTest.php

```diff
@@ -1252,6 +1341,22 @@ public function test_request_loadOpenGames() {
             "Real and dummy game lists should have matching structures");
     }
 
+    public function test_request_loadNewGames() {
+        $this->verify_login_required('loadNewGames');
+
+        $_SESSION = $this->mock_test_user_login();
+        $this->verify_invalid_arg_rejected('loadNewGames');
+
+        $args = array('type' => 'loadNewGames');
+        $retval = $this->verify_api_success($args);
+        $dummyval = $this->dummy->process_request($args);
+
+        $this->assertEquals('ok', $dummyval['status'], 'Dummy load of new games should succeed');
+    }
```

Comment:
Seems to me that `loadNewGames` is in some way the simple part of this functionality, and you're not testing it very deeply anyway --- what's really important is to have a responderTest on accepting and rejecting new games --- you could pattern the API wrapper off `verify_api_joinOpenGame()` and the test itself off `test_request_joinOpenGame()`, and that would be pretty simple and add regression-testing of the logic around accepting and rejecting games.